apps/prairielearn/src/lib/config.ts

```diff
@@ -265,6 +265,7 @@ const ConfigSchema = z.object({
   ptHost: z.string().default('http://localhost:4000'),
   checkAccessRulesExamUuid: z.boolean().default(false),
   questionRenderCacheType: z.enum(['none', 'redis', 'memory']).default('none'),
+  cacheType: z.enum(['none', 'redis', 'memory']).default('none'),
```

Comment:
I added `cacheType` to the config file. This is similar to the currently used `questionRenderCacheType` but the naming convention is less specific in nature. We will need to update this in prod configs for use.

Reply:
This sounds great. In regards to setting `config.cacheType = config.questionRenderCacheType`, I have also added a caveat that `config.cacheType` must be `none`. This would prevent accidentally overwriting `cacheType` if the user added `cacheType` to the configs without also removing the existing usage of `questionRenderCacheType`. This change was made in [85bf7c4](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/85bf7c47890d4ff51f012fcf7fe6f09ea45c002c).

Reply:
For backwards-compatibility, I think `questionRenderCacheType` should currently always take priority. We also have no way to differentiate between someone _not_ setting `cacheType` and someone setting `cacheType` to `none`, so I'm not sure this caveat behaves exactly as we'd want.

Reply:
That makes sense, I was thinking about priority of `cacheType` and `questionRenderCacheType` differently. I have removed the added conditional in [cc33414](https://github.com/PrairieLearn/PrairieLearn/pull/9360/commits/cc33414906007727a55854776b4e89552b90b0dd).